src/static/stilyagi/assets/styles/stilyagi-site.css

```
@@ -0,0 +1,505 @@
+}
+
+/* inline verdicts */
+.verdict {
+  display: inline-block;
+  font-family: var(--font-sans);
+  font-weight: 700;
+  font-size: 0.75rem;
+  letter-spacing: 0.14em;
+  text-transform: uppercase;
+  padding: 2px 8px 1px;
+  border: 1px solid currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)